torch_dipu/csrc_dipu/base/basedef.h

```diff
@@ -11,6 +11,11 @@
 #define DIPU_DEVICE_TYPE_MACRO XPU
 #define DIPU_AUTOGRAD_DEVICE_TYPE_MACRO AutogradXPU
 
+#define DeviceTypeDIPU  at::DeviceType::XPU
```

Comment:
we already have a const auto DIPU_DEVICE_TYPE = at::DeviceType::XPU;
``` suggestion
```

Reply:
have fixed it